Patches/Vanilla Bionics Expansion/HediffDefs/FSFAdvancedBionics_AddedParts.xml

```diff
@@ -148,12 +148,67 @@
 			</value>
 		</li>
 		
+		<!--Implants-->
+		
 		<!--Tone down Speed torso implant dodge chance to be more inline with CE stats (also so it doesn't instantly max out dodge)-->
 		
 		<li Class="PatchOperationReplace">
 			<xpath>Defs/HediffDef[defName="FSFImplantTorsoSpeed"]/stages/li/statOffsets/MeleeDodgeChance</xpath>
 			<value>
-					<MeleeDodgeChance>+2.5</MeleeDodgeChance>
+					<MeleeDodgeChance>1</MeleeDodgeChance>
+			</value>
+		</li>
+		
+		<!--Add carry weight buff to worker frame-->
+		
+		<li Class="PatchOperationAdd">
+			<xpath>Defs/HediffDef[defName="FSFImplantTorsoWorker"]/stages/li/statOffsets</xpath>
+			<value>
+					<CarryWeight>25</CarryWeight>
```

Comment:
This is a fairly significant carry weight boost, something in the neighborhood of +50%. This isn't unreasonable, just wanted to make sure that was the intention.

Reply:
Yep, that's the intention, as it's supposed to be gotten very late in the game (after even the advanced bionics tech from the same mod), and the upgrade itself costs a fair bit more than a suit of power armor. The sprinter frame, the competing module of the same tier outright just provides +100% moving capacity which will outright double the carry capacity if there are no other bionics.

The 25 extra weight + the worker frame's own movement capcity buff brings them to about the same performance carry weight wise without other augmentations (though the worker is slightly stronger once factoring in addtional bionic legs or such)

Reply:
Fair enough. Loads and runs with no errors on my end.